src/coffea/dataset_tools/rucio_utils.py

```diff
@@ -0,0 +1,294 @@
+import getpass
+import json
+import os
+import re
+import subprocess
+from collections import defaultdict
+
+from rucio.client import Client
+
+# Rucio needs the default configuration --> taken from CMS cvmfs defaults
+os.environ["RUCIO_HOME"] = "/cvmfs/cms.cern.ch/rucio/current"
```

Comment:
Check if it is already set, only override if not. (for our non-CMS users!)

Reply:
@valsdav